modules/widget-visibility/widget-conditions/widget-conditions.js

```diff
@@ -200,32 +200,36 @@ jQuery( function( $ ) {
 			select = condition.find( '.conditions-rule-minor' ).html( '' ),
 			major = condition.data( 'rule-major' );
 
-		if ( ! major ) {
+		if ( ! major || major === 'post_type' ) {
 			select.attr( 'disabled', 'disabled' );
 			return;
 		}
 
 		minor = condition.data( 'rule-minor' );
 		hasChildren = condition.data( 'rule-has-children' );
-		majorData = widget_conditions_data[ major ];
 
-		for ( i = 0, _len = majorData.length; i < _len; i++ ) {
-			key = majorData[i][0];
-			val = majorData[i][1];
+		if ( widget_conditions_data.hasOwnProperty(major) ) {
```

Comment:
Do you think it will be better if we remove the first condition and leave this one? It may prevent similar issues in the future if there are such, while the first one will handle `post_type` only. I wasn't sure which one is better that's why added both.